CharlieBackend.Api.UnitTes/EventServiceTest.cs

```diff
@@ -0,0 +1,176 @@
+﻿using Moq;
+using Xunit;
+using System;
+using AutoMapper;
+using FluentAssertions;
+using System.Threading.Tasks;
+using CharlieBackend.Core.Mapping;
+using CharlieBackend.Core.Entities;
+using CharlieBackend.Business.Exceptions;
+using static FluentAssertions.FluentActions;
+using CharlieBackend.Data.Repositories.Impl.Interfaces;
+using CharlieBackend.Core.DTO.Schedule;
+using CharlieBackend.Business.Services.ScheduleServiceFolder;
+
+namespace CharlieBackend.Api.UnitTest
+{
+    public class EventServiceTest : TestBase
+    {
+        private readonly IMapper _mapper;
+        private readonly Mock<IScheduledEventRepository> _eventRepositoryMock;
+        private readonly Mock<IMentorRepository> _mentorRepositoryMock;
+        private readonly Mock<IThemeRepository> _themeRepositoryMock;
+        private readonly Mock<IStudentGroupRepository> _groupRepositoryMock;
+        private readonly ScheduledEvent validEvent;
```

Comment:
Please, start names of private fields from `_` to be consistent. It gives an understanding of whether it is a class member or a local variable.